manager/integration/tests/common.py

```diff
@@ -233,6 +233,12 @@
 NODE_UPDATE_RETRY_COUNT = 30
 disk_being_syncing = "being syncing and please retry later"
 
+# customize the timeout for HDD
+disktype = os.environ['LONGHORN_DISK_TYPE']
+if disktype == "hdd":
+    RETRY_COUNTS = 300
+    DEFAULT_DEPLOYMENT_TIMEOUT = 300
```

Comment:
From the previous experiments, the timeout should be 32x longer to get rid of most of (not all of) the timeout test cases.
And there is another timeout value STREAM_EXEC_TIMEOUT should be adjusted too.

Reply:
Thank you. Fixed